securedrop/tests/test_journalist.py

```diff
@@ -427,13 +427,14 @@ def test_admin_cannot_delete_self(journalist_app, test_admin, test_journo):
 
 
 def test_admin_edits_user_password_success_response(journalist_app,
```

Comment:
one thing we'd want for merge is test coverage for the case in this PR, basically: 

1. Test admin changes test user password
2. Test user visits page, gets 200 OK and "You have been logged out due to password change" in page text
3. User is indeed actually logged out, i.e. they are redirected to login upon trying to access another page, e.g. `assert "Login to access" in text`